src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -540,7 +540,7 @@ case class JesBackend(actorSystem: ActorSystem)
 
       status match {
         case Run.Success(events) if backendCall.runtimeAttributes.failOnStderr && stderrLength.intValue > 0 =>
-          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength")).future
+          FailedExecutionHandle(new Throwable(s"${log.tag} execution failed: stderr has length $stderrLength"), returnCode.toOption).future
```

Comment:
So this will just yield a `None` if the RC file is undownloadable or unparseable as an int?  I think that's the behavior we want, but maybe throw in a comment to this effect as perhaps it's not super obvious?